payments-service/subscriptions-delete.js

```diff
@@ -63,7 +63,7 @@ export const handler = (event, context, callback) => {
   cancelSubscription(id, provider)
     .then(resp => {
       logOperation(id, provider);
-      callback(null, ok({ cors: true, body: data }));
+      callback(null, ok({ cors: true, body: event.data }));
```

Comment:
@osahyoun I swapped `data` for `event.data` because data was undefined - was this what you had in mind or was this meant to do something else?

Reply:
Thanks, @Tuuleh. That's exactly what I had in mind.